cmd/sops/main.go

```diff
@@ -176,13 +185,18 @@ func main() {
 					Action: func(c *cli.Context) error {
 						pgpFps := c.StringSlice("pgp")
 						kmsArns := c.StringSlice("kms")
+						azkvs := c.StringSlice("azkv")
 						var group sops.KeyGroup
 						for _, fp := range pgpFps {
 							group = append(group, pgp.NewMasterKeyFromFingerprint(fp))
 						}
 						for _, arn := range kmsArns {
 							group = append(group, kms.NewMasterKeyFromArn(arn, kms.ParseKMSContext(c.String("encryption-context"))))
 						}
+						// NOTE: Why isn't GCP here?
```

Comment:
I was confused by GCP not being present here - is there some reasoning for when it is needed and when not?